conans/client/command.py

```diff
@@ -569,10 +574,10 @@ def imports(self, *args):
                             "execution. e.j: conan imports ./imported_files --undo ")
         parser.add_argument("--file", "-f", help="Use another filename, "
                             "e.g.: conan imports -f=conanfile2.py")
-        parser.add_argument("-d", "--dest",
+        parser.add_argument("--import-folder", "--import_folder", "-if",
```

Comment:
AS ``-if`` is install-folder in other context, I will keep the same here, and call ``import_folder`` something like ``-imf``